README.md

```diff
@@ -42,15 +121,99 @@ Install requirements:
 npm install
 ```
 
-## Getting started
+#### Setup environments
 
-### How to make a deposit
+Being a NPM package, the SDK itself doesn’t use environment variables with exception of the tests. But in order to use the SDK you will need to set up an environment file following a specific .env.example file.
+Env.example
 
-```bash
-npm run-script eg:deposit
+##### Locally (Ganache)
+
+Create .env.ganache file
+
+```
+APP_BLOCKCHAIN_WEBSOCKET_URL=ws://localhost:8546
+APP_CLIENT_API_URL=http://localhost:8080
+APP_ETH_PRIVATE_KEY=0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e
+APP_NIGHTFALL_MNEMONIC=
+APP_TOKEN_ADDRESS_1=0x4f3c4F8D4575Cf73c2FAf9F36cc505e19E65B9C0
 ```
-WIP, explain env file
 
+##### Goerli Testnet
+
+Create .env.goerli file
+
+```
+APP_BLOCKCHAIN_WEBSOCKET_URL=wss://goerli.infura.io/ws/v3/89114afb34fb4dc29d333f9849679c53
+APP_CLIENT_API_URL=http://localhost:8080
+APP_ETH_PRIVATE_KEY=
+APP_NIGHTFALL_MNEMONIC=
+APP_TOKEN_ADDRESS_1=0x499d11E0b6eAC7c0593d8Fb292DCBbF815Fb29Ae
+```
+
+##### EthereumMainnet
+
+### Getting started
+
+To get a good idea of how to interact with Nightfall using the SDK, there are a set of example scripts that allow for a better understanding of the core features of Nightfall.
+
+All of the scripts are explained in short detail below and are to be run in a local environment.
+
+\*Note
+The scripts load an .env file, it is suggested to use different files per environment, for example creating `examples/.env.ganache` for a local environment, based on `.env.example`.
+Upon each running of any of the scripts a new instance of the User class is created, if an existing Mnemonic isn’t specified in the `.env` file, a new Mnemonic is assigned to that specific user.
```

Comment:
SUGGESTION :arrow_down: 

To get a good idea of how to interact with Nightfall using the SDK, there is a set of example scripts in `/examples` that allow for a better understanding of the core features of Nightfall.

All of the scripts are explained in short detail below. You can try them out using the given commands or you can refer to the package.json `scripts`. E.g.:

```
"eg:deposit:ganache": "ts-node -r dotenv/config examples/txDeposit.ts dotenv_config_path=./examples/.env.ganache dotenv_config_debug=true"
```

#### Environment setup

Being a NPM package, the SDK doesn't use environment variables with exception of the tests and logs. However, **you need to pass certain parameters which we recommend to keep private**. As such, the example scripts will use a `config` object that preloads env vars from a file. We suggest to use different files per environments. The deposit script above will look for an `/examples/.env.ganache` file (based on `.env.example`).

```
# Contents of .env.ganache
APP_BLOCKCHAIN_WEBSOCKET_URL=ws://localhost:8546
APP_CLIENT_API_URL=http://localhost:8080
APP_ETH_PRIVATE_KEY=0x4775af73d6dc84a0ae76f8726bda4b9ecf187c377229cb39e1afa7a18236a69e
APP_NIGHTFALL_MNEMONIC=
APP_TOKEN_ADDRESS_1=0x4f3c4F8D4575Cf73c2FAf9F36cc505e19E65B9C0
```

#### Nightfall keys

Upon each running of any of the scripts a new instance of the User class is created, if an existing Mnemonic isn’t specified in the `.env` file, a new Mnemonic is assigned to that specific user.

_Etc._

Let me know :pray:

Reply:
Makes more sense yes, not having goreli and mainnet SDK variables setup allows for this approach and it is much cleaner.